src/app/vm/vm-actions/vm-actions-component/vm-access.component.ts

```diff
@@ -52,8 +52,13 @@ export class VmAccessComponent {
     return this.sshAccessService.getPassword(vm) || this.translateService.instant('VM_POST_ACTION.NOT_SET');
   }
 
-  public getSSHKeys(keys: string): string[] {
-    const sshKeys = keys.split(' ');
-    return sshKeys;
+  public isValidUrl(url: string): boolean {
```

Comment:
We already have a [tested function ](https://github.com/bwsw/cloudstack-ui/blob/master/src/app/shared/utils/isUrl.ts)